libvast/builtins/commands/rebuild.cpp

```diff
@@ -143,13 +143,13 @@ struct run {
 /// The interface of the REBUILDER actor.
 using rebuilder_actor = system::typed_actor_fwd<
   // Start a rebuild.
-  caf::reacts_to<atom::start, start_options>,
+  auto(atom::start, start_options)->caf::result<void>,
```

Comment:
Why `auto` + `->` instead of putting `result<T>` to the front?

Reply:
Practical reason: I did this with three calls to sed, and I couldn't figure out how to do both arguments and return value in one go.

I must say I do also find this much more readable as it aligns better with the lambdas in the behavior set.

Reply:
Sed sets the coding standards, great. 😅

If you think it's more readable, I'm fine with it. Personally, I think we could reduce the number of characters. I would argue that it should look like a `std::function<...>` declaration where people put the return value first.

Reply:
Honestly, I'd prefer using trailing return types everywhere. It's enforceable with clangd and you can even automatically transform to it using clangd's suggested actions. The scoping for trailing return types is much more natural than for leading return types. May actually be something to just enforce globally in the future to stay up to date with the times.